Platform: win32, unknown when installing C# in VS Code on Windows 10

## Environment data
`dotnet --info` output:
.NET Command Line Tools (1.0.0-preview2-1-003177)

Product Information:
 Version:            1.0.0-preview2-1-003177
 Commit SHA-1 hash:  a2df9c2576

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64

VS Code version: 1.8.1
C# Extension version: 1.6.2

## Steps to reproduce
Install C# in VS Code and Open a .cs file.

## Expected  behavior
C# extension ahould recognize my operating system and the Omnisharp server should load without errors.

## Actual behavior
C# log displays...
```
Updating C# dependencies...
Platform: win32, unknown
```

OmniSharp log displays...
OmniSharp server started
    Path: ~\.vscode\extensions\ms-vscode.csharp-1.6.2\bin\omnisharp\OmniSharp.exe
    PID: 9920

[ERROR] Error: OmniSharp server load timed out. Use the 'omnisharp.projectLoadTimeout' setting to override the default delay (one minute).

-------------------------

There's only one other issue here that relates to mine https://github.com/OmniSharp/omnisharp-vscode/issues/1056 and that seemed to have been resolved by a missing path. I already have that path and when I run `wmic os get osarchitecture`, I get...
OSArchitecture
64-bit

Not sure what else to try to fix this.

--- a/src/platform.ts
+++ b/src/platform.ts
@@ -167,27 +167,14 @@ export class PlatformInformation {
     }
 
     private static GetWindowsArchitecture(): Promise<string> {
-        return util.execChildProcess('wmic os get osarchitecture')
-            .then(architecture => {
-                if (architecture) {
-                    let archArray: string[] = architecture.split(os.EOL);
-                    if (archArray.length >= 2) {
-                        let arch = archArray[1].trim();
-
-                        // Note: This string can be localized. So, we'll just check to see if it contains 32 or 64.
-                        if (arch.indexOf('64') >= 0) {
-                            return "x86_64";
-                        }
-                        else if (arch.indexOf('32') >= 0) {
-                            return "x86";
-                        }
-                    }
-                }
-
-                return unknown;
-            }).catch((error) => {
-                return unknown;
-            });
+        return new Promise<string>((resolve, reject) => {
+            if (process.env.PROCESSOR_ARCHITECTURE === 'x86' && process.env.PROCESSOR_ARCHITEW6432 === undefined) {
+                resolve('x86');
+            }
+            else {
+                resolve('x86_64');
+            }
+        });
     }
 
     private static GetUnixArchitecture(): Promise<string> {

